profitbricks/client.py

```diff
@@ -200,14 +203,14 @@ def update_datacenter(self, datacenter_id, **kwargs):
 
         return response
 
-    # FirewallRule Functions
+    # Firewall Rule Functions
 
     def get_firewall_rule(self, datacenter_id,
                           server_id, nic_id, firewall_rule_id):
         """
-        Retrieves a single FirewallRule by ID.
+        Retrieves a single Firewall Rule by ID.
```

Comment:
Does FirewallRule refer to the object name or not? In the latter case, please use lower case.